src/nvim/api/buffer.c

```diff
@@ -1400,6 +1400,9 @@ Array nvim_buf_get_extmarks(Buffer buffer, Integer ns_id,
 ///                   callbacks. The mark will only be used for the current
 ///                   redraw cycle, and not be permantently stored in the
 ///                   buffer.
+///               - gravity : the direction the extmark will be shifted in
+///                   when new text is inserted. Must be either 'left' or
+///                   'right'.
```

Comment:
there should also be `end_gravity` for completeness (last arg to `marktree_put_pair`)